map/routing_manager.cpp

```diff
@@ -702,6 +710,14 @@ bool RoutingManager::InsertRoute(Route const & route)
           FillTurnsDistancesForRendering(segments, subroute->m_baseDistance, subroute->m_turns);
           break;
         }
+      case RouterType::Helicopter:
+        {
+          subroute->m_routeType = df::RouteType::Helicopter;
+          subroute->m_headFakeDistance = -1.0f;
+          subroute->m_tailFakeDistance = 90.0f; //Assuming that Helicopter line is shorter than 90°. Otherwise line tail would have a gray color.
```

Comment:
nit: Put this long comment above the line.

Reply:
Fixed